internal/exportimport/model/importkeys.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package model
+
+import (
+	"crypto/ecdsa"
+	"crypto/x509"
+	"encoding/pem"
+	"errors"
+	"fmt"
+	"time"
+)
+
+type ImportFilePublicKey struct {
+	ExportImportID int64
+	KeyID          string
+	KeyVersion     string
+	PublicKeyPEM   string
+	From           time.Time
+	Thru           *time.Time
+}
+
+func (pk *ImportFilePublicKey) PublicKey() (*ecdsa.PublicKey, error) {
+	block, _ := pem.Decode([]byte(pk.PublicKeyPEM))
+	if block == nil || block.Type != "PUBLIC KEY" {
```

Comment:
I don't know of block.Type is guaranteed to be `PUBLIC KEY`

Reply:
I can only find ECDSA examples w/ PUBLIC KEY, but it's also safe to remove this since we're checking the type later